### ISSUE TITLE
Vulkan: Raster occlusion margin is too big on baked occluders

### ISSUE THREAD

**Issue by tinmanjuggernaut** [godotengine/godot#63771]
### Godot version

v4.0.alpha.custom_build.d29e95687

### System information

Win 10/64 NVIDIA GeForce GTX 1060/PCIe/SSE2

### Issue description

Simple scene w/ baked occluders

![image](https://user-images.githubusercontent.com/632766/182136917-da538bf1-1039-42ab-ae1f-7a5648e0a74a.png)

The background meshes do not occlude properly. Here you can see even at the start of the video the back ones should be hidden as soon as they are fully occluded, such as the left one as it begins. However, I have to move the camera until it is nearly inside the front object before the back ones turn off. But once you do go inside, the back objects turn on. So the camera is a few centimeters from the object before they turn off.

https://user-images.githubusercontent.com/632766/182137576-e34e0f0f-dd05-4ec1-9c16-2d042fc6e4a3.mp4

Here I made the front object 10x the size and pulled it back so the objects didn't overlap. Now the left back object is on, even though it's fully occluded. This is the border. If I move the camera just a hair right, the left object will turn off. 

![image](https://user-images.githubusercontent.com/632766/182138398-0f26d240-1b4a-4556-a7bc-840f07d8e248.png)

![image](https://user-images.githubusercontent.com/632766/182138626-4a615dba-dc6b-4a89-91f6-9809a24f6745.png)

### Steps to reproduce

Look at the MRP in overdraw mode.

### Minimal reproduction project

[test_occbake.zip](https://github.com/godotengine/godot/files/9233543/test_occbake.zip)

**Comment by Calinou** [comment]
Occludees use their AABB for occlusion culling, so sphere-like objects will have a lot of wasted space by design.

**Comment by tinmanjuggernaut** [comment]
Hmm, so in the video and image above, if I mentally draw a cube around the bg rocks, there's still an excessive margin around the aabb.

**Comment by Calinou** [comment]
> Hmm, so in the video and image above, if I mentally draw a cube around the bg rocks, there's still an excessive margin around the aabb.

You can visualize the AABB by selecting one of the rocks at a time in the editor.

**Comment by tinmanjuggernaut** [comment]
Ok. Here I've selected the background rocks and you can see that even considering their rotated AABBs, the margin is still wayyyyy too big. There's no reason at all for that middle rock to not be culled, especially.

![image](https://user-images.githubusercontent.com/632766/182327635-fcbae9a9-6f2b-4faf-ab34-86608db4d431.png)

**Comment by Calinou** [comment]
This may be due to DirectionalLight3D shadow rendering causing the AABBs to be extended (to prevent occludees' shadows from disappearing when they shouldn't). To disable the preview sun in the editor, try adding a DirectionalLight3D node then hiding it.

**Comment by tinmanjuggernaut** [comment]
Good idea. I tried it, but the light made no difference to the effective culling margin. In the MRP there's no ground, so limited shadow use anyway.

**Comment by myaaaaaaaaa** [comment]
See https://github.com/godotengine/godot/issues/53288#issuecomment-1455233730
